# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../djvulibre/gcc-4.0.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- djvulibre-3.5.14/libdjvu/GString.h_orig	2005-11-15 19:18:22.000000000 +0100
+++ djvulibre-3.5.14/libdjvu/GString.h	2005-11-15 19:16:55.000000000 +0100
@@ -140,6 +140,10 @@
 # endif
 #endif
 
+class GBaseString;
+class GUTF8String;
+class GNativeString;
+
 // Internal string representation.
 class GStringRep : public GPEnabled
 {
